src/functions/env.ts

```diff
@@ -198,11 +198,10 @@ function findEnvfiles(
   const files: string[] = [".env"];
   if (isEmulator) {
     files.push(FUNCTIONS_EMULATOR_DOTENV);
-  } else {
-    files.push(`.env.${projectId}`);
-    if (projectAlias && projectAlias.length) {
-      files.push(`.env.${projectAlias}`);
-    }
+  }
+  files.push(`.env.${projectId}`);
+  if (projectAlias) {
+    files.push(`.env.${projectAlias}`);
   }
```

Comment:
> Contents of .env.local take precedence over .env and the project-specific .env file.

With what you're doing here, `.env.<project-id-or-alias>` would override `.env.local`.

Reply:
🤦🏼‍♂️ That was sloppy of me. Thanks for the catch!